addon/styles/_frost-link.scss

```diff
@@ -214,10 +215,69 @@ $frost-color-link-focus-box-shadow: 0 1px 6px rgba(0,158,239, 0.75);
     }
 
     .svg {
-      margin-left: 10px;
+      &:not(.action) {
+        margin-left: 10px;
+      }
       .frost-icon {
         vertical-align: middle;
       }
     }
   }
+
+  // Action button styling
+  &.action {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    width: 75px;
+    height: 75px;
+    background-color: $frost-color-white;
```

Comment:
Let's alias these colors (e.g. $frost-link-info-bar-action-background) to make them more themeable